project/script_VMRCM.tcl

```diff
@@ -91,12 +45,15 @@ foreach i $modules_to_test {
   # Define FPGA, clock frequency & common HLS settings.
   source settings_hls.tcl
 
+  puts [join [list $::env(CCFLAG) ] ""]  
+
   # run C-simulation
-  csim_design -mflags "-j8"
+  csim_design -mflags "-j1"
```

Comment:
@aryd For consistency with the other modules, could you please revert this?
```suggestion
  csim_design -mflags "-j8"
```

Reply:
Done. (I'm not sure where these changes in the VMRCM comes from - as far as I recall I have not touched this module...)